packages/framework/ai-collab/src/test/explicit-strategy/integration.spec.ts

```diff
@@ -0,0 +1,187 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-internal-modules
+import { createIdCompressor } from "@fluidframework/id-compressor/internal";
+// eslint-disable-next-line import/no-internal-modules
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
+import {
+	SchemaFactory,
+	SharedTree,
+	TreeViewConfiguration,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/tree/internal";
+
+import { generateTreeEdits } from "../../explicit-strategy/index.js";
+
+import { initializeOpenAIClient } from "./utils.js";
+
+const sf = new SchemaFactory("Planner");
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Session extends sf.object("Session", {
+	id: sf.identifier,
+	title: sf.string,
+	abstract: sf.string,
+	sessionType: sf.required(sf.string, {
+		metadata: {
+			description:
+				"This is one of four possible strings: 'Session', 'Workshop', 'Panel', or 'Keynote'",
+		},
+	}),
+	created: sf.required(sf.number, {
+		metadata: {
+			// llmDefault: () => Date.now(), TODO: Add this back when we have a defaulting value solution
+		},
+	}),
+	lastChanged: sf.required(sf.number, {
+		metadata: {
+			// llmDefault: () => Date.now(), TODO: Add this back when we have a defaulting value solution
+		},
+	}),
+}) {}
+
+const SessionType = {
+	session: "Session",
+	workshop: "Workshop",
+	panel: "Panel",
+	keynote: "Keynote",
+};
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Sessions extends sf.array("Sessions", Session) {}
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Day extends sf.object("Day", {
+	sessions: sf.required(Sessions, {
+		metadata: {
+			description: "The sessions scheduled on this day.",
+		},
+	}),
+}) {}
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Days extends sf.array("Days", Day) {}
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Conference extends sf.object("Conference", {
+	name: sf.string,
+	sessions: sf.required(Sessions, {
+		metadata: {
+			description:
+				"These sessions are not scheduled yet. The user (or AI agent) can move them to a specific day.",
+		},
+	}),
+	days: Days,
+	sessionsPerDay: sf.number,
```

Comment:
Nit: I think this property doesn't make much sense conceptually (in the test data it says 2, when day one has 4 sessions and day two has 3), so I'd lean towards removing it to avoid confusion.

Reply:
Removed that property from the schema, and renamed this file to conferece.spec.ts under the /integration path